doc/GettingStarted.md

```diff
@@ -245,6 +246,16 @@ In addition, Vitess requires the software and libraries listed below.
     If your machine requires a proxy to access the Internet, you will need
     to set the usual environment variables (e.g. `http_proxy`,
     `https_proxy`, `no_proxy`).
+    
+    Note:
+        On OS X, if you have already installed grpc and protobuf using homebrew, you need to unlink protobuf while the ./bootstrap.sh
+        script is running.
+        
+        ```sh
+        brew unlink grpc/grpc/google-protobuf 
```

Comment:
Does it make sense to put these two commands in bootstrap.sh instead?

That should be more user friendly than adding these steps to the documentation.